.github/workflows/tests.yml

```diff
@@ -91,17 +91,22 @@ jobs:
       if: failure() && github.repository_owner == 'deepset-ai' && github.ref == 'refs/heads/main'
 
   unit-tests:
-    name: Unit / ${{ matrix.os }}
+    name: Unit / ${{ matrix.topic }} / ${{ matrix.os }}
     needs:
       - mypy
       - pylint
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest]
```

Comment:
Typo: `macos-latest]` to `macos-latest`. I think this is causing the failing test.